### PR TITLE
fix typos and incorrect variable names in manual

### PR DESCRIPTION
**Pull request by eugenemedvediev**
Fixed typos in manual

### CODE REVIEW

File: src/content/Effects.js [hunk 1]
```diff
@@ -145,13 +145,13 @@ def foreach(f: a ~> Unit, l: List[a]): Unit & Impure`}
 
                 <p>
                     Note that here <Code>ef1</Code> and <Code>ef2</Code> are arbitrary boolean variables
-                    which are not directly associated with the effect of <Code>f</Code> or
```
Comment (eugenemedvediev) [inline src/content/Effects.js:148]: ah, looks like this issue was always addressed. Should I rollback this space changes?

Comment (magnus-madsen) [reply src/content/Effects.js:148]: Sorry, what happened?

Comment (eugenemedvediev) [reply src/content/Effects.js:148]: Screenshot from 2022-01-03 13-55-09
Here you can see that `f or g` became `f org`. The same problem with `effects of g(`.
I thought that extra space will fix it, but looks like there already space present.

Comment (eugenemedvediev) [reply src/content/Effects.js:148]: I have this problem on iPhone SE 2. Issue is reproducible using firefox's device simulation view.

Comment (magnus-madsen) [reply src/content/Effects.js:148]: OK. No idea unfortunately.

**Comment by magnus-madsen** [comment]
Do you want me to merge this PR as-is?

File: src/content/Effects.js (hunk 1 above)
Comment (mlutze) [reply src/content/Effects.js:148]: I think if we make a change so that `<Code>xyz</Code>` is not at the end/beginning of a line, that should fix it.

Comment (eugenemedvediev) [reply src/content/Effects.js:148]: I don't know why `Code` tag is rendered without spaces around. I don't know how to build this project to try to fix it. Should I revert this extra spaces changes, so we can merge all other fixes?

Comment (magnus-madsen) [reply src/content/Effects.js:148]: OK, let us do that then!

**Comment by magnus-madsen** [comment]
Thanks @eugenemedvediev 

Will be online soon.

**Pull request merged**